src/src/SelfServiceApiClient.js

```diff
@@ -342,9 +400,15 @@ export class SelfServiceApiClient {
     }
 
     const accessToken = await getSelfServiceAccessToken();
-    const response = await callApi(link.href, accessToken, "POST", {
-      capabilityId: capabilityId,
-    });
+    const response = await callApi(
```

Comment:
We have a few of these changes (or similar). 
Partly this is because of the order of the parameters, but I somehow like that we are specific about what we pass along.
So even though it looks like a lot of boilerplate, I might be quite okay with it 👍